### PR TITLE
Candidate 2.x.x

### PR DESCRIPTION
**Pull request by GordonSmith**
## Checklist:
- [ ] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit message includes a "fixes" reference if appropriate.
  - [ ] The commit is signed.
- [ ] The change has been fully tested:
  - [ ] I have viewed all related gallery items
  - [ ] I have viewed all related dermatology items
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised new issues to address them separately

### CODE REVIEW

**Comment by GordonSmith** [comment]
@jbrundage can you review these upmerged changes from 1.20.x?

File: packages/common/src/Utility.ts [hunk 1]
```diff
@@ -275,7 +275,7 @@ export function faCode(key) {
 
 //  Template   ---
 //  https://github.com/Matt-Esch/string-template (MIT)
-const nargs = /\{([0-9a-zA-Z_\[\]]+)\}/g;
+    var nargs = /\{([0-9a-zA-Z_\s\[\]]+)\}/g;
```
Comment (GordonSmith) [inline packages/common/src/Utility.ts]: Should be const

**Comment by jbrundage** [comment]
Will do

File: .vscode/launch.json [hunk 2]
```diff
@@ -4,8 +4,8 @@
         {
             "type": "chrome",
             "request": "launch",
-            "name": "Roxie DDL",
-            "url": "file:///${workspaceRoot}/demos/dermatology.html?src/marshaller/HTML.roxie",
+            "name": "Dermatology",
+            "url": "file:///${workspaceRoot}/demos/dermatology.html",
```
Comment (jbrundage) [inline .vscode/launch.json]: This should be `"url": "file:///${workspaceRoot}/demos/dermatology/index.html",`

Comment (GordonSmith) [reply .vscode/launch.json]: I think your reviewing the wrong commit - as this is not in the final PR...

**Comment by jbrundage** [comment]
The changes look good

**Pull request merged**